mne/viz/_3d.py

```diff
@@ -560,9 +552,9 @@ def _plot_mri_contours(mri_fname, surf_fnames, orientation='coronal',
 @verbose
 def plot_alignment(info, trans=None, subject=None, subjects_dir=None,
                    surfaces='head', coord_frame='head',
-                   meg=None, eeg='original',
+                   meg=None, eeg='original', ren=None,
                    dig=False, ecog=True, src=None, mri_fiducials=False,
-                   bem=None, seeg=True, show_axes=False, fig=None,
```

Comment:
@GuillaumeFavelier be careful with API breakage. We have a strong policy of deprecation warnings for one release cycle. Any code that works with 0.17 should work with 0.18 out of the box but with warnings in case of deprecation.

Reply:
I can rollback `_3d.py` and think about a slower strategy of integration.